build/int.cloudbuild.yaml

```diff
@@ -71,6 +71,66 @@ steps:
 - id: appinfra-teardown
```

Comment:
You probably want to move all the appinfra*-teardowns to after the final appinfra*-verify (similar to the earlier stages), otherwise their won't be a point where everything can be tested holistically.

Reply:
https://pantheon.corp.google.com/cloud-build/builds/a5697301-d70e-41d1-8dac-b46d2acd1096;step=35?mods=-local_coliseum&project=cloud-foundation-cicd

timed out at 2hr at step 35: appinfratransactionhistory-verify, hence increased the CI timeout to 3hr.

each app is independent, it should be ok for each app to init/apply/verify/teardown sequentially.

Reply:
Sounds Good - Was thinking it might be nice to include an E2E test that confirms that the whole stack works together, but that can wait for later.